src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -640,7 +640,16 @@ namespace ZoneEquipmentManager {
 				CalcZoneSizing( CurOverallSimDay, ControlledZoneNum ).HeatMassFlow = 0.0;
 				CalcZoneSizing( CurOverallSimDay, ControlledZoneNum ).HeatZoneTemp = 0.0;
 				CalcZoneSizing( CurOverallSimDay, ControlledZoneNum ).HeatZoneHumRat = 0.0;
-			}
+			} else {
+				CalcZoneSizing( CurOverallSimDay, ControlledZoneNum ).CoolLoad = 0.0;
+				CalcZoneSizing( CurOverallSimDay, ControlledZoneNum ).CoolMassFlow = 0.0;
+				CalcZoneSizing( CurOverallSimDay, ControlledZoneNum ).CoolZoneTemp = Node( ZoneNode ).Temp;
+				CalcZoneSizing( CurOverallSimDay, ControlledZoneNum ).CoolZoneHumRat = ZoneSizingInput( ControlledZoneNum ).CoolDesHumRat;
```

Comment:
@mjwitte I pulled the latest of 90946736 wshp #4567 branch but I see many variables inaccessible error flag in ZoneEquipmentManager code. My be I should start from develop again. Was there any conflict when you merged in develop?